### PR TITLE
Jupiter terminal: adjust terminal position for mobile

### PR DESCRIPTION
**Pull request by dicksonpys**
This PR adjust the Terminal's position so that it isn't blocked by the nav header. And more accessible on landscape mode.

From:
image
image

To:
image
image

### CODE REVIEW

File: hub/components/RealmHeader/index.tsx [hunk 1]
```diff
@@ -84,25 +84,7 @@ async function importJupiter() {
...
-      el.onerror = rej;
-      el.id = 'jupiter-load-styles';
-      el.rel = 'stylesheet';
-      el.href = 'https://terminal.jup.ag/main.css';
-      document.head.append(el);
-    }
-  });
-
-  return Promise.all([script, css]).then(() => {
+  return Promise.all([script]).then(() => {
```
Comment (GregoryNEUT) [inline hub/components/RealmHeader/index.tsx:87]: nit: no need for `Promise.all` anymore now that there is only one promise

**Review by GregoryNEUT — COMMENTED** [review]
Hello, just a nit :) hope it helps

File: hub/components/RealmHeader/index.tsx (hunk 1 above)
Comment (dicksonpys) [reply hub/components/RealmHeader/index.tsx]: this is done!

Comment (dicksonpys) [reply hub/components/RealmHeader/index.tsx]: also, I've noticed that user's aren't actually auto connected on refresh some times, I've also pushed a conditional that only does passThrough when it is connected.
```tsx
passThroughWallet: connected ? wallet : undefined,
```

**Pull request merged**